network.go

```diff
@@ -1184,6 +1290,31 @@ func isPhysicalIface(ifaceName string) (bool, error) {
 	return true, nil
 }
 
+// Long term, this should be made more configurable.  For now matching path
+// provided by CNM VPP plugin
+const hostSocketSearchPath = "/tmp/vhostuser_%s/vhu.sock"
+
+// isVhostuserIface checks if an interface is a dummy placeholder
+// for a vhost-user socket
+func isVhostuserIface(netInfo NetworkInfo) (bool, string, error) {
+	if netInfo.Iface.Name == "lo" {
+		return false, "", nil
+	}
+
+	// for now, check for socket file existence at known location.
+	// Identify this be searching for a file in a path which contains
+	// the interfaces IP address.  ie:
+	//  /tmp/vhostuser_<ipv4 address>/vhu.sock
```

Comment:
Here we should explain where the `/tmp/vhostuser_ipv4/vhu.sock` pattern is coming [from](https://github.com/clearcontainers/vpp/pull/5/files).

Reply:
Sure.  Will do.